lib/vanagon/platform/dsl.rb

```diff
@@ -405,6 +405,20 @@ def zypper_repo(definition)
       def add_build_repository(*args)
         @platform.add_build_repository(*args)
       end
+
+      # Adds an override to a build requirement. When component.build_requires is
+      # called, the component will check to see if the requirement matches one of
+      # these overrides, and use the override if there's a match.
+      #
+      # @param build_requirement [String] build requirement to overrides
```

Comment:
(nitpick) 'overrides' -> 'override'

Reply:
(If `Overrides` is being used as a plurality, I can live with that)